local-cli/cliEntry.js

```diff
@@ -15,6 +15,7 @@ const defaultConfig = require('./default.config');
 const dependencies = require('./dependencies/dependencies');
 const generate = require('./generate/generate');
 const library = require('./library/library');
+const link = require('./rnpm/link/src/link');
```

Comment:
The idea is that we make rnpm a core part of React Native and eventually stop publishing the standalone rnpm module completely.

What would be the advantage of using dev-dependencies? Would it help with bundle size?

In general, we have dependencies in React Native's package.json (e.g. yeoman) that are only needed by the CLI and not the apps. But since the apps don't require these, we don't bundle and ship them as part of the apps, right?